sw/device/tests/sim_dv/sram_ctrl_scrambled_access_test.c

```diff
@@ -6,6 +6,7 @@
 #include <stdint.h>
```

Comment:
can you rename the test to `sram_ctrl_ret_scrambled_access_test` to align with the test *main_scrambled*